index.js

```diff
@@ -0,0 +1,106 @@
+const { default: axios } = require('axios');
+const { Handler } = require('htmlmetaparser');
+const { Parser } = require('htmlparser2');
+
+/**
+ * This is a recursive function that returns an array of dataset site URLs.
+ * If the URL supplied is a data catalog collection, it takes all the part collections in hasPart and crawls them.
+ * If the URL supplied is a data catalog, it takes the dataset array and flattens them. 
```

Comment:
Suggest that if the `dataCatalogUrl` param is not supplied, it should default to `'https://openactive.io/data-catalogs/data-catalog-collection.jsonld'` (the "official" OpenActive data catalog). This should help prevent the official data catalog URL from being hard-coded downstream.

Reply:
Yep makes sense